spring-integration-smb/src/main/java/org/springframework/integration/smb/filters/SmbRegexPatternFileListFilter.java

```diff
@@ -48,4 +51,19 @@ protected String getFilename(SmbFile file) {
 		return (file != null ? file.getName() : null);
 	}
 
+	/**
+	 * Indicates whether the file is a directory or not.
+	 * @param file SMB file object
+	 * @return true if it's a directory.
+	 * @see AbstractDirectoryAwareFileListFilter#isDirectory(java.lang.Object)
```

Comment:
I think we don't need JavaDocs on this method at all. It is inherited.

More over, please, don't use imports if they are mentioned only in JavaDocs.